### ISSUE TITLE
Deep learning 

### ISSUE THREAD

**Issue by jskim19** [GDSC-Ewha-4th/Study-paperReview#22]
Deep learning 
Yann LeCun, Yoshua Bengio & Geoffrey Hinton 

### Deep learning
Key aspect : layers of features are not designed by human engineers 
Image recognition, speech recognition, analysing particle accelerator data 등에 적용 가능 

### Supervised Learning 
error를 측정하는 objective function 계산 
Weight vector 조정하기 위해 gradient vector 계산 
대부분은 SGD(stochastic gradient descent) 사용 

### Backpropagation to train multilayer architectures 
모듈의 출력에 대해 backward로 그래디언트 계산
Pre-training - initialize the weights of a deep network 

### Convolutional neural networks 
Multiple array 형태로 들어온 data를 처리 하기 위함 

### Image Understanding with deep convolutional networks
ConvNets는 detection, segmentation, recognition에 좋은 성과를 보임 
이미지는 픽셀 레벨로 labelling됨 
다양한 회사에서 스마트폰, 카메라, 로봇과 자율주행을 위해 개발중 

### Distributed representations and language processing 
Generalization to new combinations of the values of learned features 
Potential for another exponential advantage 

### The future of deep learning
Natural language understanding, combining deep learning and reinforcement

**Comment by minha62** [comment]
정리가 깔끔해서 이해하기 좋았습니다! 좋은 내용 감사합니다:)
